tests/functional/adapter/test_unit_testing.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+
+from dbt.tests.adapter.unit_testing.test_types import BaseUnitTestingTypes
+from dbt.tests.adapter.unit_testing.test_case_insensitivity import BaseUnitTestCaseInsensivity
+from dbt.tests.adapter.unit_testing.test_invalid_input import BaseUnitTestInvalidInput
+
+
+class TestUnitTestingTypesDuckDB(BaseUnitTestingTypes):
```

Comment:
@jwills I pulled these in (selfishly), since it seemed like unit testing might "just work" on `dbt-duckdb` with the v1.8 interface updates.

It's all well & good on the Main Duck, but it looks like the test is failing on BuenaVista for the last data type in the list (`ARRAY[1,2,3]`):
```
  File "/home/runner/work/dbt-duckdb/dbt-duckdb/.tox/buenavista/lib/python3.9/site-packages/buenavista/postgres.py", line 104, in <lambda>
    BVType.INTEGERARRAY: (1007, lambda v: "{" + ",".join(v) + "}"),
TypeError: sequence item 0: expected str instance, int found
sequence item 0: expected str instance, int found
---------------------- CSV report: buenavista_results.csv ----------------------
=========================== short test summary info ============================
FAILED tests/functional/adapter/test_unit_testing.py::TestUnitTestingTypesDuckDB::test_unit_test_data_type - AssertionError: unit test failed when testing model with ARRAY[1,2,3]
```
How would you feel about me adding `@pytest.mark.skip_profile("buenavista")` to this test? I know that adds a bit of tech debt / mystery for you later on

Reply:
I think that’s just because I don’t properly support array types in the BV Postgres serialization, so it’s perfectly understandable and okay to mark it as skipped